### PR TITLE
[TW-790] Add  topic on what happens when resource limits are reached

### PR DESCRIPTION
**Pull request by mark-anthony-postman**
- Added new "About resource usage" page under "Administration"
- Add links to "About resource usage" page on other pages that mention resource usage
- Removed references to serverless usage

### CODE REVIEW

File: src/pages/docs/administration/resource-usage.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Integrations
+
+Your Postman plan gives you a limited number of [integrations](/docs/integrations/intro-integrations/) for connecting Postman to third-party tools you use for API development. After the limit is reached, you won't be able to add a new integration. You will get a notification in the Postman app when attempting to add a new integration.
+
+<img alt="Integrations limit reached" src="https://assets.postman.com/postman-docs/v10/usage-error-integrations-v10.jpg" width="361px"/>
+
+To add a new integration, delete one of your existing integrations or [upgrade your plan](/docs/administration/billing/#team-and-plan-changes).
+
+## Cloud Agent usage
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/administration/resource-usage.md:83]: What do you think about adding your note "`> To learn more about the resources included with your Postman plan and what happens when you reach your usage limits, go to [About resource usage](/docs/administration/resource-usage/).`" to the Postman Cloud Agent section at `/docs/getting-started/about-postman-agent/#the-postman-cloud-agent`?

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks great! Thanks!

File: src/pages/docs/administration/resource-usage.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Postman Basic, Professional, and Enterprise plans give you a limited number of [custom domains](/docs/publishing-your-api/custom-doc-domains/) you can use when publishing API documentation. After the limit is reached, you won't be able to add a new custom domain.
+
+![Custom domains limit reached](https://assets.postman.com/postman-docs/v10/usage-error-custom-domains-v10.jpg)
+
+To add a new custom domain, delete one of your existing custom domains, [purchase an add-on](/docs/administration/billing/#purchasing-add-ons), or [upgrade your plan](/docs/administration/billing/#team-and-plan-changes).
+
+## Storage usage
+
+Your Postman plan gives you a limited amount of storage for [uploaded images](/docs/publishing-your-api/authoring-your-documentation/#uploading-an-image) in your API documentation. You will get an error message when attempting to upload an image that would exceed your storage limit.
```
Comment (HeatherStenson) [inline src/pages/docs/administration/resource-usage.md:59]:
```suggestion
Your Postman plan gives you a limited amount of storage for [uploaded images](/docs/publishing-your-api/authoring-your-documentation/#uploading-an-image) in your API documentation. You will get an error message if you try to upload an image that would exceed your storage limit.
```

**Review by HeatherStenson — APPROVED** [review]
Looks great! 🍊

File: src/pages/docs/administration/resource-usage.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/resource-usage.md:83]: Great idea! Done! 😀

File: src/pages/docs/administration/resource-usage.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+* [Mock usage](#mock-usage)
+* [Monitoring usage](#monitoring-usage)
+* [Custom domains](#custom-domains)
+* [Storage usage](#storage-usage)
+* [Postman API usage](#postman-api-usage)
+* [Integrations](#integrations)
+* [Cloud Agent usage](#cloud-agent-usage)
+
+## Mock usage
```
[the remark this answers is not in the document]
Comment (mark-anthony-postman) [reply src/pages/docs/administration/resource-usage.md:31]: I was following the UI here, but I think you're right it's better to follow the terminology we us in the LC.

File: src/pages/docs/administration/resource-usage.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    name: "Additional Resources"
+  - type: link
+    name: "Postman Plans and Pricing"
+    url: "https://www.postman.com/pricing/"
+  - type: link
+    name: "Resource Usage dashboard"
+    url: "https://go.postman.co/billing/add-ons/overview"
+---
+
+Postman provides you with a limited number of resources you can use each month, depending on your Postman plan. Monthly resources include calls to the Postman API, requests made by monitors and scheduled collection runs, mock server requests, and Cloud Agent requests. Your Postman plan also has other limits, such as the number of integrations you can create, the number of custom domains, and storage for uploaded images.
```
Comment (jkonrath-postman) [inline src/pages/docs/administration/resource-usage.md:15]: The first "limited" sounds a bit... unenthusiastic - like Postman is a limited tool. Maybe "given" or "specified"?

**Review by jkonrath-postman — APPROVED** [review]
LGTM! I had one nitpicky suggestion on wording, but otherwise this is a great addition!

File: src/pages/docs/administration/resource-usage.md (hunk 4 above)
Comment (mark-anthony-postman) [reply src/pages/docs/administration/resource-usage.md:15]: It's hard to make limits sound fun 😄 I'll go with "specified"

**Pull request merged**